src/pages/sighting/identification/buildMatchingSetQuery.js

```diff
@@ -0,0 +1,58 @@
+import { flattenDeep, get } from 'lodash-es';
+
+function addChildMatches(rootRegion, matchId, insideRegion) {
+  const matchFound = rootRegion?.id === matchId;
+  const insideRegionNew = matchFound || insideRegion;
+  const parentMatch = insideRegionNew ? [rootRegion] : [];
+  const children = rootRegion?.locationID || [];
```

Comment:
Same here... these look like specific kinds of children (locationIdChildren?).

Reply:
Oh! That's right... locationId is the name of recursive nodes in locationId.json. I'm guessing that this stems from that.

Reply:
I guess maybe they could still have a more descriptive name than children. 

May I suggest a 500 character name 😈 ?

Reply:
childMatches?

Reply:
would descendants be a better word? children does have another meaning in the context of react. matches doesn't make total sense to me because they aren't necessarily matches, just the children of the region that is actively being looked at